websites/management/commands/rename_by_text_id.py

```diff
@@ -0,0 +1,30 @@
+from django.conf import settings
+from django.core.management import BaseCommand
+from django.utils.text import slugify
+
+from gdrive_sync.api import rename_file
+
+
+class Command(BaseCommand):
+    """Rename the file on S3 associated with the WebsiteContent object to a new filename, and update the object and corresponding DriveFile."""
+
+    help = __doc__
+
+    def add_arguments(self, parser):
+
+        parser.add_argument(
+            "--text_id",
+            dest="text_id",
+            help="text_id of WebsiteContent object to be updated",
+            required=True,
+        )
+
+        parser.add_argument(
+            "--new_filename",
+            dest="new_filename",
+            help="New filename to be associated with the WebsiteContent object",
+            required=True,
+        )
+
+    def handle(self, *args, **options):
+        rename_file(options["text_id"], options["new_filename"])
```

Comment:
`sys.stdout.write("File successfully renamed")`

Reply:
Updated in 8eee92a8b1ec916f8fd918512194b6dcb7152963.